components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/editor/js/source-editor/constants.js

```diff
@@ -105,5 +106,9 @@ define(function () {
         TOKEN_TOOLTIP_POINT_RECOGNITION_COMPLETION: "TOKEN_TOOLTIP_POINT_RECOGNITION_COMPLETION"
     };
 
+    constants.asyncAPI = {
```

Comment:
We have a dedicated constant file right ? shall we move this there

Reply:
Seems each component has one of its own
https://github.com/wso2/carbon-analytics/blob/a60e7f7f072b19e2c52f3c3367539e3b9142c837/components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/editor/js/source-editor/constants.js